radixdlt-java/src/test/java/com/radixdlt/client/core/crypto/RadixECKeyPairsTestJUnit5.java

```diff
@@ -0,0 +1,52 @@
+package com.radixdlt.client.core.crypto;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertThat;
+
+public class RadixECKeyPairsTestJUnit5 {
+
+    @BeforeEach
+    public void beforeSuite() {
+        // Ensure the BouncyCastle providers are loaded into memory
+        // (because BouncyCastle SHA-256 is used in "seed" tests).
+        ECKeyPairGenerator.install();
+    }
+
+    @Test
+    @DisplayName("Given privateKey1 and privateKey2 When we compare them Then they should be different")
```

Comment:
Can we break these up over multiple lines?

Reply:
I don't believe so but come one... you should be happy, look at all those spaces you can add!

Reply:
Just a note though, to break it up, an inner class using @Nested annotation could be used and you can place the @DisplayName("Given privateKey1 and privateKey2") there on top of the inner class and keeping @DisplayName("When we compare them Then they should be different") on the method inside the inner class which will make it shorter and look nicer.

This is used when there are multiple methods using the same "Given". In this case though, there is only one method so not really necessary but would make it look neater for sure.

Reply:
It's possible to use string concatenation in annotations in Java, so could be something like:
```
@DisplayName(
    "Given privateKey1 and privateKey2 " +
    "When we compare them " +
    "Then they should be different")
```